thirdparty/SemanticUI-Angular/examples/accordion/directives.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
```

Comment:
Do we have to bring in all the examples?

Reply:
No, it's just a clone if their git repo. I had to do a one-line change to their stuff :( I can delete them if you want.